pkg/ccl/backupccl/restore_online.go

```diff
@@ -409,6 +409,23 @@ func (r *restoreResumer) doDownloadFiles(ctx context.Context, execCtx sql.JobExe
 	if err := grp.Wait(); err != nil {
 		return errors.Wrap(err, "failed to generate and send download spans")
 	}
+	if err := r.cleanupAfterDownload(ctx, details); err != nil {
+		return err
+	}
+	return nil
```

Comment:
Nit: `return r.cleanupAfterDownload(ctx, details)` is all you need

Reply:
Good point. Done.